docs/FAQ.md

```diff
@@ -0,0 +1,224 @@
+
+### What is the difference between .NET Interactive and Jupyter?
+
+The .NET Interactive kernel (i.e. the `dotnet-interactive` [.NET tool](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools)) is a Jupyter-compliant kernel that can be used with Project Jupyter just like any other kernel. Like other Jupyter kernels, .NET Interactive isn't opinionated about which frontend you use. 
+
+The Polyglot Notebooks extension for VS Code isn't required to use the .NET Interactive kernel, but it does provide access to some additional functionality that isn't typical of Jupyter frontends, such as the ability to switch languages (i.e. subkernels) on a per-cell basis.
+
+### What is a kernel?
+
+A kernel is an execution engine for a notebook. While the notebook UI (or "frontend") is responsible for displaying text, code, and execution results, the kernel is where the actual processing of the code takes place. Kernels are UI-agnostic. They typically run in a different process from the UI, and can often run on a different machine.
+
+In .NET Interactive, there might be several kernels within a single kernel process. We refer to these kernels as "subkernels." Each one represents a stateful unit of computation with a set of capabilities that can include running code, sharing variables, and providing language services (e.g. code completions, diagnostic squiggles, inline help) for a specific language. A kernel in .NET Interactive need not be in its own process. Multiple .NET Interactive subkernels can work together in a single notebook session whether they share a process or are distributed across multiple machines.
+
+### What is a Jupyter kernel?
+
+A Jupyter kernel is any kernel that implements the [Jupyter Message Protocol (JMP)](https://jupyter-client.readthedocs.io/en/stable/messaging.html#). The most commonly-used Jupyter kernel is [IPython](https://en.wikipedia.org/wiki/IPython), an interactive shell for Python, from which Project Jupyter grew. When started in Jupyter mode (using the command line `dotnet interactive jupyter`), .NET Interactive is a Jupyter kernel.
+
+### What is a frontend?
+
+The term "frontend" is often used to describe the UI for a notebook editor. Jupyter Notebook, JupyterLab, nteract Desktop, and the [Jupyter Extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) for VS Code and GitHub Codespaces are examples of notebook frontends.
+
+### What is a subkernel?
+
+A subkernel is a concept in .NET Interactive that describes one among many kernels within a single notebook session. Subkernels correspond to the different languages available in a Polyglot Notebook. Subkernels can be added dynamically at runtime, for example when connecting to a data source or a kernel running in a remote process.
+
+### What is a proxy kernel?
```

Comment:
Like the section above, it would be really nice if there was an example we could call out here too. Maybe that could be deferred to when we have the python kernel as well.

Reply:
Python isn't a proxy kernel. But any kernel can be proxied. The most common example currently is the JavaScript kernel. I can add an example explaining that.